tests/llmobs/test_experiments.py

```diff
@@ -186,3 +193,82 @@ def test_experiment_create(llmobs, test_dataset):
     )
     assert exp_id is not None
     assert exp_run_name.startswith("test_experiment")
+
+
+@pytest.mark.parametrize(
+    "test_dataset_records",
+    [
+        [
+            DatasetRecord(input_data={"prompt": "What is the capital of France?"}, expected_output={"answer": "Paris"}),
+            DatasetRecord(
+                input_data={"prompt": "What is the capital of Canada?"}, expected_output={"answer": "Ottawa"}
+            ),
+        ]
+    ],
+)
+def test_experiment_run_task(llmobs, test_dataset, test_dataset_records):
+    exp = llmobs.experiment("test_experiment", dummy_task, test_dataset, [dummy_evaluator], project_name="test-project")
+    task_results = exp._run_task(1, raise_errors=False)
+    assert len(task_results) == 2
+    assert task_results[0] == {
+        "idx": 0,
+        "output": {"prompt": "What is the capital of France?"},
+        "metadata": {
+            "timestamp": mock.ANY,
+            "duration": mock.ANY,
+            "dataset_record_index": 0,
+            "experiment_name": "test_experiment",
+            "dataset_name": "test-dataset-test_experiment_run_task[test_dataset_records0]",
+            "span_id": mock.ANY,
+            "trace_id": mock.ANY,
```

Comment:
nit: we can cut down on the length of the tests (and make them a bit more readable) with

```suggestion
            k: mock.ANY for k in ("timestamp", "duration", ...)
```

Reply:
rip unfortunately this doesn't work in python 3.8 lol